TrafficCapture/KafkaPrinter/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
```

Comment:
Can you update this to 8.0.2 I don't think jdk 17 of your dockerfile image will work with this earlier version